usaspending_api/common/query_with_filters.py

```diff
@@ -119,7 +119,10 @@ def _generate_elasticsearch_query(cls, filter_values: List[dict], query_type: _Q
             agency_name = v["name"]
             agency_tier = v["tier"]
             agency_type = v["type"]
+            toptier_name = v.get("toptier_name")
             agency_query = ES_Q("match", **{f"{agency_type}_{agency_tier}_agency_name__keyword": agency_name})
+            if agency_tier == "subtier" and toptier_name is not None:
```

Comment:
Does this also need to be applied to `matview_filters.py` and `sub_award.py` where we filter for Postgres queries?

Reply:
You mean [this](https://github.com/fedspendingtransparency/usaspending-api/blob/8790aea5c447bbfedffd86f5f840a63d1b94a8cf/usaspending_api/awards/v2/filters/matview_filters.py#L147-L152)?  Interestingly, it was already there.  I guess we had intended to support this functionality at one point... or maybe we did.  I don't know.  Feel free to double check it, although I didn't write it so it'll probably work.

Reply:
Yep that is it. Overlooked that when I checked.